compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -1905,3 +1905,6 @@ error.tuple.and.expression.size.does.not.match=\
 
 error.infer.size.only.supported.in.the.first.dimension=\
   inferred array size is only allowed in the first dimension of an array type descriptor
+
+error.named.arg.not.allowed.included.in.included.record.param=\
+  named arg ''{0}'' is not allowed since it is a filed in ''{1}'' which is an arg
```

Typo - filed -> field.

Like I mentioned previously, error messages need to be as complete and clear as possible, since these are for users. We should use the full form `argument`.

---

The error is still not clear IMO. It is not that the named argument is not allowed, it's that you can only specify one of the two.

I would use something like

```suggestion
  duplicate argument for ''{0}'' via named argument and argument for included record parameter
```

---

Please update the diagnostic and the key also to better reflect the error.

---

Changed